armi/physics/neutronics/crossSectionSettings.py

```diff
@@ -109,6 +157,14 @@ def __init__(
         self.mergeIntoClad = mergeIntoClad
         self.fileLocation = fileLocation
         self.meshSubdivisionsPerCm = meshSubdivisionsPerCm
+        if self.geometry is not None or self.fileLocation is not None:
```

Comment:
why only validate in this condition? Why not always validate?

Reply:
Updated in revision.